Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py

```diff
@@ -1710,6 +1933,224 @@ def report_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
     )
 
 
+def dictionary_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Retrieve dictionary configuration details.
+
+    Args:
+        client (Client): Cisco ESA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: Readable outputs for XSOAR, including dictionary configuration details.
+    """
+    mode = args.get("mode", DEFAULT_MODE_DICTIONARIES)
+    host_name = args.get("host_name", "")
+    group_name = args.get("group_name", "")
+    dictionary_name = args.get("dictionary_name", "")
+
+    host_name, group_name = check_dictionary_mode_args(mode, host_name, group_name)
+    response = client.dictionary_list_request(dictionary_name=dictionary_name,
+                                              mode=mode,
+                                              host_name=host_name,
+                                              group_name=group_name
+                                              )
+
+    if dictionary_name:
+        name = f'Information for Dictionary: {dictionary_name}'
+    else:
+        name = f'Information for All Configured Dictionaries in mode: {mode}'
+
+    readable_output = tableToMarkdown(
+        name=name,
+        t=response.get('data'),
+        removeNull=True,
+    )
+
+    return CommandResults(
+        outputs_prefix="CiscoESA.Dictionary",
+        outputs=response.get('data'),
+        raw_response=response,
+        readable_output=readable_output
+    )
+
+
+def dictionary_add_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Add a new dictionary configuration.
+
+    Args:
+        client (Client): Cisco ESA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: Readable outputs for XSOAR, indicating the successful addition of the dictionary.
+    """
+    mode = args.get("mode", DEFAULT_MODE_DICTIONARIES)
+    host_name = args.get("host_name", "")
+    group_name = args.get("group_name", "")
+    dictionary_name = args.get("dictionary_name", "")
+    whole_words = args.get("whole_words", "")
+    words = args.get("words", "")
+    ignore_case_sensitive = args.get("ignore_case_sensitive", "")
+
+    host_name, group_name = check_dictionary_mode_args(mode, host_name, group_name)
+    ignore_case_sensitive = int(bool(ignore_case_sensitive))
+    whole_words = int(bool(whole_words))
+    words = convert_words_to_list(words)
```

Comment:
You can just use `argToList` from CommonServerPython
No need for this helper function

Reply:
argToList doesn't convert it the right way. When words = "[['test1'], ['test2']]" (as the api should get), argToList convert it to ["[['test1']", "['test2']]"].